scss/os/_overrides_bootstrap.scss

```diff
@@ -156,3 +156,10 @@ ul {
     margin-bottom: 0;
   }
 }
+
+//sass-lint:disable no-duplicate-properties, no-vendor-prefixes
+input:-webkit-autofill,
+input:-webkit-autofill:active  {
+  -webkit-box-shadow: 0 0 0 30px #555 inset !important;
```

Comment:
These should use Bootstrap variables so they work in any theme, instead of assuming the dark theme.

Reply:
Bootstrap variables "$input-bg" and "$input-color" have been added